src/components/tipRecords/TipRecord.vue

```diff
@@ -30,22 +30,45 @@
         <TipTitle :tip-title="tip.title" />
       </div>
       <TipPreview
+        v-if="tipUrl"
         :tip="tip"
         :go-to-tip="goToTip"
         :tip-url="tipUrl"
-      />
-      <div class="tip__footer">
-        <div class="tip__footer_wrapper">
-          <div
-            class="tip__comments"
-            :class="[{ 'tip__comments--hascomments': tip.commentCount }]"
+      >
+        <TipInput
+          v-if="tip.type !== 'PostWithoutTip'"
+          :tip="tip"
+        />
+      </TipPreview>
+      <div
+        class="tip-footer"
+        @click.stop
+      >
+        <TipInput
+          v-if="tip.type === 'PostWithoutTip'"
+          :post="tip"
+        />
+        <div class="actions-wrapper">
+          <ButtonPlain
+            class="actions"
+            :class="{ active: isTipPinned }"
+            @click="pinOrUnPinTip"
```

Comment:
shouldn't we remove pin/unpin from the dropdown menu then?

Reply:
@onvisions could you share your opinion?

Reply:
I think it is not confusing in this case to have "Pin to Channel" both in the drop-down and as a "star"-icon UI element.
Star icon is not only an element triggering an action but also element showing the state of the item (pinned/unpinned to user's channel). Star should be "empty" for not-pinned and full for pinned to channel items (posts).
1. In case the dropdown extended options menu is not displayed user has the option to pin directly clicking the Star.
2. In case the pin action is triggered by the drop down the Star also changes its state from emty to full star so it's pretty clear that the item is also a visualization of the state of the post - pinned/unpinned.